### ISSUE TITLE
Double quotation marks detection issue

### ISSUE THREAD

**Issue by ParadiseDS** [jfcherng/Sublime-ShellScriptImproved#40]
ENV: macOS Sierra 10.12.2, sublime 3 build 3143, just install ShellScriptImproved through Package Control.

In some cases, the shell syntax can only detect one of the Double quotation marks in a pair, which result in code content displayed as string. See the pics below.

 
<img width="673" alt="2017-12-21 2 50 04" src="https://user-images.githubusercontent.com/3424318/34243961-4e547ac8-e65e-11e7-9984-9a0610aadf2b.png">

code

```shell
function sendgsm() {
    for mobile in $MOBILELIST
    do
        $GSM_CMD $mobile@"$* - from $(hostname)"
    done
}

function alert() {
    echo "$*" | sendmail "$*"
    sendgsm "$*"
}

: ${SUPERVISE_MODULE_NAME:="supervise.${PRO_NAME}"}

: ${PID_FILE:="none"}

function gen_pidfile() {
    local config_file=$1

    [[ -f ${config_file} && `grep "pid" ${config_file}` ]] && {
        PID_FILE=`grep "pid" ${config_file} | awk -F':' '{print $2}'`
        notice "Getting pid_file : ${PID_FILE} succeed!"
        return 0
    }

    fatal "Getting pid_file unexpected, please do check ${CONFIG_FILE} !"
}
```

AFAIK, the official shell syntax also has double quotation marks detection issue, but the bad case are different from the one above, showed as below.

<img width="592" alt="2017-12-21 2 51 25" src="https://user-images.githubusercontent.com/3424318/34244005-76d8b090-e65e-11e7-9a87-8cfbf03d538d.png">

**Comment by rwols** [comment]
Sublime Text build 3145 and onward should fix your issue for the "official" syntax.

**Comment by jfcherng** [comment]
Should be fixed in `1.2.26`.

You could also checkout the `v3145` release of https://github.com/sublimehq/Packages as mentioned by rwols. Not sure whether later versions have compatibility issue with ST 3143 or not.

**Comment by ParadiseDS** [comment]
Fix confirmed. Thanks for the quick fix!

> You could also checkout the v3145 release of https://github.com/sublimehq/Packages as mentioned by rwols.

I may check it out in the next release build, as currently ShellScriptImproved can meet my needs completely.
